wildfly/connectors/amazon/feature-pack-amazon/src/main/resources/modules/layers.conf

```diff
@@ -0,0 +1 @@
+layers=dv
```

Comment:
Does this file need to be repeated in every feature pack? Ultimately we just need a single `layers.conf` in the modules directory at the top level. I'm not familiar with Galleon so I might be missing something.

Reply:
When you do the build you will find only one copy of the layers.conf file in the deployment. It is needed for each feature as the features can be installed individually using Galleon if the administrator is trying to pare down the deployment.

Reply:
Could the Wildfly Integration feature pack just install it instead? Wouldn't an admin always have to install that if they wanted to use any of the connectors?

https://github.com/Doug-Webster-Certara/teiid/tree/master/wildfly/teiid-feature-pack/wildfly-integration-feature-pack/src/main/resources/modules

It looks like it was previously in the modules directory in the `resources.original` directory.

https://github.com/Doug-Webster-Certara/teiid/tree/master/wildfly/teiid-feature-pack/wildfly-integration-feature-pack/src/main/resources.original/modules

Reply:
I see what you mean. Each of the connector feature packs has a common dependency of the teiid-client-galleon-pack. I will leave the layers.conf file in the teiid-client-galleon-pack but remove it from the individual connector feature packs.